parent/pom.xml

```diff
@@ -669,6 +669,58 @@
       </build>
     </profile>
 
+		<profile>
+			<id>deploy-to-jboss.org</id>
+			<build>
+				<plugins>
+					<plugin>
+						<artifactId>maven-dependency-plugin</artifactId>
+						<executions>
+							<execution>
+								<goals>
+									<goal>unpack</goal>
+								</goals>
+								<phase>deploy</phase>
+								<configuration>	
+									<artifactItems>
+										<artifactItem>
+					          					<groupId>org.jboss.tools.releng</groupId>		
+											<artifactId>jbosstools-releng-publish</artifactId>
+											<version>4.3.0.Alpha2-SNAPSHOT</version>
+											<type>tar.gz</type>
+											<outputDirectory>${project.build.directory}/releng-scripts</outputDirectory>	
```

Comment:
We can/should put a guard in the rsync.sh script to avoid such errors.
The best thing would be to be able to conditionally trigger the mojos based on packaging-type, but that's an utopia.